week-1-practise/level-2/1-tax.js

```diff
@@ -5,7 +5,9 @@
   Sales tax is 20% of the price of the product
 */
 
-function calculateSalesTax() {}
+function calculateSalesTax(price) {
+  return .2*price;
```

Comment:
Stylistically, it is preferred to have empty spaces around mathematical operators. Makes it more readable. `.2 * price`

Reply:
Ok, thanks!